Dockerfile

```diff
@@ -353,6 +353,7 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
 FROM binary-base AS build-cross
 ARG DOCKER_CROSSPLATFORMS
 RUN --mount=type=cache,target=/root/.cache/go-build \
+    --mount=type=tmpfs,target=/go/src/github.com/docker/docker/autogen \
```

Comment:
does order matter here? i.e, should this one be last?

Reply:
It seems like the list gets sorted in buildkit, but I wet ahead and made it shortest to longest.